rust/repositories.bzl

```diff
@@ -206,7 +206,8 @@ def load_arbitrary_tool(ctx, tool_name, param_prefix, tool_subdirectory, version
     # N.B. See https://static.rust-lang.org/dist/index.html to find the tool_suburl for a given
     # tool.
     tool_suburl = produce_tool_suburl(tool_name, target_triple, version, iso_date)
-    url = "https://static.rust-lang.org/dist/{}.tar.gz".format(tool_suburl)
+    static_rust = ctx.os.environ["STATIC_RUST_URL"] if "STATIC_RUST_URL" in ctx.os.environ else "https://static.rust-lang.org"
+    url = "{}/dist/{}.tar.gz".format(static_rust, tool_suburl)
```

Comment:
For some reason I can't comment on unchanged lines, but can you tweak the function comment (specifically where it says files are sourced from static.rust-lang.org? I guess just adding "or STATIC_RUST_URL set with action_env" would be fine.

Reply:
Added the corresponding comment.

Also answer to your question from the other thread, I'll be happy to be added as collaborator to rules_rust.

Reply:
I actually don't seem to be able to update contributor permissions. [According to Github docs](https://help.github.com/articles/permission-levels-for-a-user-account-repository/#collaborator-access-on-a-repository-owned-by-a-user-account), a Repository Owner is the only one with such permissions. We probably need someone from the bazelbuild org to do this...

A cursory search of our internal issues landed me on some [Github Admin issue](http://b/68970165) with your name on it. I suspect you might know better than me how to get collaborator status set up for yourself. I've already added you to cargo-raze (as well as @mfarrugi).

Reply:
I used to be the admin of the bazelbuild org but since I left the Bazel team I am no longer. I guess @philwo is still one as he replaced me as a org owner when I left.

@acmcarther you should actually be an admin, usually the Googler who is maintaining the repository is admin of the repo (because of Google's policy, only Googlers can be admin of repo on Google owned github org).

Reply:
/cc @helenalt @jhfield since they seem to be the new owners now and philwo is OOO: to summarize, @acmcarther is not admin on the repo so cannot give me write access. 

Historically, we were giving admin access to googlers who owner a repo (and non-googler cannot be admin according to restrictions on Google's org).

Reply:
Damien: you should be set now. dslomov@ changed the perms last night.